assets/js/components/surveys/CurrentSurvey.stories.js

```diff
@@ -16,128 +16,106 @@
-		} );
+SurveyMultipleQuestionsStory.args = TemplateArgs( multiQuestionSurvey );
 
-		registry
-			.dispatch( CORE_USER )
-			.receiveTriggerSurvey( fixtures.multiQuestionSurvey, {
-				triggerID: 'storybookSurvey',
-			} );
-		registry.dispatch( CORE_USER ).receiveGetTracking( { enabled: true } );
-	},
-};
+export const SurveyMultipleQuestionsConditionalStory = Template.bind( {} );
+SurveyMultipleQuestionsConditionalStory.storyName = 'Conditional';
+SurveyMultipleQuestionsConditionalStory.args = TemplateArgs(
+	multiQuestionConditionalSurvey
+);
 
 export const SurveyNotAnsweredNoFollowUpStory = Template.bind( {} );
 SurveyNotAnsweredNoFollowUpStory.storyName = 'New survey (no follow-up CTA)';
-SurveyNotAnsweredNoFollowUpStory.args = {
-	setupRegistry: ( registry ) => {
-		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, {
-			body: {},
-			status: 200,
-		} );
-
-		registry
-			.dispatch( CORE_USER )
-			.receiveTriggerSurvey(
-				fixtures.singleQuestionSurveyWithNoFollowUp,
-				{ triggerID: 'storybookSurvey' }
-			);
-		registry.dispatch( CORE_USER ).receiveGetTracking( { enabled: true } );
-	},
-};
+SurveyNotAnsweredNoFollowUpStory.args = TemplateArgs(
+	singleQuestionSurveyWithNoFollowUp
+);
 
 export const SurveyAnsweredPositiveStory = Template.bind( {} );
 SurveyAnsweredPositiveStory.storyName = 'Completed';
-SurveyAnsweredPositiveStory.args = {
-	setupRegistry: ( registry ) => {
-		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, {
-			body: {},
-			status: 200,
-		} );
-
-		registry
-			.dispatch( CORE_USER )
-			.receiveTriggerSurvey( fixtures.singleQuestionSurvey, {
-				triggerID: 'storybookSurvey',
-			} );
-		registry.dispatch( CORE_USER ).receiveGetTracking( { enabled: true } );
-
+SurveyAnsweredPositiveStory.args = TemplateArgs(
+	singleQuestionSurvey,
+	( registry ) => {
 		registry
 			.dispatch( CORE_FORMS )
-			.setValues(
-				`survey-${ fixtures.singleQuestionSurvey.session.session_id }`,
-				{
-					answers: [
-						{
-							question_ordinal: 1,
-							answer: {
-								answer: { answer_ordinal: 5 },
-							},
+			.setValues( `survey-${ singleQuestionSurvey.session.session_id }`, {
+				answers: [
+					{
+						question_ordinal: 1,
+						answer: {
+							answer: { answer_ordinal: 5 },
 						},
-					],
-				}
-			);
-	},
-};
+					},
+				],
+			} );
+	}
+);
 
 export const SurveyWithTermsStory = Template.bind( {} );
 SurveyWithTermsStory.storyName = 'With Terms';
-SurveyWithTermsStory.args = {
-	setupRegistry: ( registry ) => {
-		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, {
-			body: {},
-			status: 200,
-		} );
-
-		registry
-			.dispatch( CORE_USER )
-			.receiveTriggerSurvey( fixtures.singleQuestionSurvey, {
-				triggerID: 'storybookSurvey',
-			} );
+SurveyWithTermsStory.args = TemplateArgs(
+	singleQuestionSurvey,
+	( registry ) => {
 		registry.dispatch( CORE_USER ).receiveGetTracking( { enabled: false } );
-	},
-};
+	}
+);
```

Comment:
Per last comment, this would be a normal `setupRegistry` with the tracking option disabled `setupCurrentSurvey( registry, singleQuestionSurvey, { trackingEnabled: false } )`

Reply:
Updated.